### PR TITLE
Create a comment in exist PR

### PR DESCRIPTION
**Pull request by AlekseyLeshko**
Hi!
Thank you, for your helpful and faster action!

I found a bug with creating a comment in existing PR.
This PR was fixing this bug.

### CODE REVIEW

File: __tests__/add-pr-comment.test.ts [hunk 1]
```diff
@@ -86,6 +88,44 @@ describe('add-pr-comment action', () => {
     await expect(run()).resolves.not.toThrow()
   })
 
+  it('creates a comment in an existing PR', async () => {
```
Comment (AlekseyLeshko) [inline __tests__/add-pr-comment.test.ts:91]: The test for a case

File: __tests__/add-pr-comment.test.ts [hunk 2]
```diff
@@ -86,6 +88,44 @@ describe('add-pr-comment action', () => {
...
+      if (key === 'comment-created') {
+        expect(value).toBe('true')
+      }
+
+      return originalSetOutput(key, value)
+    })
+
+    issueNumber = apiResponse.result[0].number
+    nock('https://api.github.com')
+      .get(`/repos/${repoFullName}/commits/${commitSha}/pulls`)
```
Comment (AlekseyLeshko) [inline __tests__/add-pr-comment.test.ts:117]: 🍒 Getting PRs

File: add-pr-comment.ts [hunk 3]
```diff
@@ -4,7 +4,7 @@ import {HttpClient} from '@actions/http-client'
 import {Endpoints, RequestHeaders} from '@octokit/types'
 import {Octokit} from '@octokit/rest'
 
-type ListCommitPullsResponse = Endpoints['GET /repos/:owner/:repo/commits/:commit_sha/pulls']['response']
+type ListCommitPullsResponse = Endpoints['GET /repos/:owner/:repo/commits/:commit_sha/pulls']['response']['data']
```
Comment (AlekseyLeshko) [inline add-pr-comment.ts:7]: Fixing the response type

File: add-pr-comment.ts [hunk 4]
```diff
@@ -39,7 +39,7 @@ const listCommitPulls = async (params: ListCommitPullsParams): Promise<ListCommi
 }
 
 const getIssueNumberFromCommitPullsList = (commitPullsList: ListCommitPullsResponse): number | null =>
-  commitPullsList.data && commitPullsList.data.length ? commitPullsList.data[0].number : null
+  commitPullsList.length ? commitPullsList[0].number : null
```
Comment (AlekseyLeshko) [inline add-pr-comment.ts:42]: 🍓 Important line

File: tsconfig.json [hunk 5]
```diff
@@ -10,7 +10,8 @@
     "noUnusedParameters": true,
     "noImplicitAny": true,
     "removeComments": false,
-    "preserveConstEnums": true
+    "preserveConstEnums": true,
+    "resolveJsonModule": true
```
Comment (AlekseyLeshko) [inline tsconfig.json:14]: For using a `json` file with stub data from github server

File: __tests__/add-pr-comment.test.ts [hunk 6]
```diff
@@ -5,12 +5,13 @@ import * as github from '@actions/github'
 import {WebhookPayload} from '@actions/github/lib/interfaces'
 import nock from 'nock'
 import run from '../add-pr-comment'
+import apiResponse from '../docs/sample-pulls-api-response.json'
```
Comment (AlekseyLeshko) [inline __tests__/add-pr-comment.test.ts:8]: Use the response stub

**Comment by mshick** [comment]
So the [`.result`](https://github.com/mshick/add-pr-comment/blob/master/add-pr-comment.ts#L38) property from `@actions/http-client` is actually the data, from an Octokit response, not the Octokit response object? Let me see if I can recreate this just to verify.

Thanks very much for highlighting and the comprehensive PR!

**Comment by mshick** [comment]
Awesome, confirmed. PR looks good. Thanks again!

**Comment by mshick** [comment]
I've re-released v1 with your fix in place. Please let me know if you encounter any other issues.

**Pull request merged**